cstool/cstool_bpf.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+
+#include <capstone/capstone.h>
+#include <capstone/platform.h>
+
+static char * ext_name[] = {
+	[BPF_EXT_LEN] = "#len",
+};
+
+void print_insn_detail_bpf(csh handle, cs_insn *ins)
+{
+	cs_bpf *bpf;
+	cs_regs regs_read, regs_write;
+	uint8_t regs_read_count, regs_write_count;
+
+	// detail can be NULL on "data" instruction if SKIPDATA option is turned ON
+	if (ins->detail == NULL)
+		return;
+
+	bpf = &(ins->detail->bpf);
+
+	unsigned i;
```

Comment:
please put all variable declaration at the beginning of code block, but no in the middle.